homeassistant/components/zone/__init__.py

```diff
@@ -120,7 +147,8 @@ def async_active_zone(
         if zone_dist is None:
             continue
 
-        within_zone = zone_dist - radius < zone.attributes[ATTR_RADIUS]
+        # Within zone if location circle is inside zone
+        within_zone = zone_dist + radius < zone.attributes[ATTR_RADIUS]
```

Comment:
Do we want to test for `<` or `<=` here?
Since it's floating point maths it's essentially the same for all practical purposes, but maybe one or the other is easier to reason about?